x-pack/legacy/plugins/siem/public/components/timeline/properties/index.tsx

```diff
@@ -170,175 +153,48 @@ export class Properties extends React.PureComponent<Props, State> {
-                    createTimeline={createTimeline}
-                    onClosePopover={this.onClosePopover}
-                    timelineId={timelineId}
-                  />
-                </EuiFlexItem>
-
-                <EuiFlexItem grow={false}>
-                  <OpenTimelineModalButton />
-                </EuiFlexItem>
-
-                <EuiFlexItem grow={false}>
-                  <InspectButton
-                    queryId={timelineId}
-                    inputId="timeline"
-                    inspectIndex={0}
-                    isDisabled={!isDataInTimeline}
-                    onCloseInspect={this.onClosePopover}
-                    show={true}
-                    title={i18n.INSPECT_TIMELINE_TITLE}
-                  />
-                </EuiFlexItem>
-
-                {width < showNotesThreshold ? (
-                  <EuiFlexItem grow={false}>
-                    <NotesButton
-                      animate={true}
-                      associateNote={associateNote}
-                      getNotesByIds={getNotesByIds}
-                      noteIds={noteIds}
-                      showNotes={this.state.showNotes}
-                      size="l"
-                      text={i18n.NOTES}
-                      toggleShowNotes={this.onToggleShowNotes}
-                      toolTip={i18n.NOTES_TOOL_TIP}
-                      updateNote={updateNote}
-                    />
-                  </EuiFlexItem>
-                ) : null}
-
-                {width < showDescriptionThreshold ? (
-                  <EuiFlexItem grow={false}>
-                    <DescriptionPopoverMenuContainer>
-                      <Description
-                        description={description}
-                        timelineId={timelineId}
-                        updateDescription={updateDescription}
-                      />
-                    </DescriptionPopoverMenuContainer>
-                  </EuiFlexItem>
-                ) : null}
-              </EuiFlexGroup>
-            </EuiPopover>
-          </EuiFlexItem>
-
-          {title != null && title.length
-            ? usersViewing.map(user => (
-                // Hide the hard-coded elastic user avatar as the 7.2 release does not implement
-                // support for multi-user-collaboration as proposed in elastic/ingest-dev#395
-                <HiddenFlexItem key={user}>
-                  <EuiToolTip
-                    data-test-subj="timeline-action-pin-tool-tip"
-                    content={`${user} ${i18n.IS_VIEWING}`}
-                  >
-                    <Avatar data-test-subj="avatar" size="s" name={user} />
-                  </EuiToolTip>
-                </HiddenFlexItem>
-              ))
-            : null}
-        </PropertiesRight>
+        <PropertiesLeft
+          isFavorite={isFavorite}
+          timelineId={timelineId}
+          updateIsFavorite={updateIsFavorite}
+          showDescription={width >= showDescriptionThreshold}
```

Comment:
i appreciate how moving the `width` threshold check up to this component eliminates the need to propagate `width` to children, since `width` updates when the timeline is resized 👍